style.css

```diff
@@ -0,0 +1,68 @@
+h1 {
```

Comment:
many styles here and layouts could have been bootstrap classes
i.e. card, card-body, rounded, or shadow-sm